notebooks/utils.py

```diff
@@ -379,3 +371,137 @@ def train_classifier(inputs, targets):
     confuson_matrix = confusion_matrix(targets_val, pred)
 
     return custom_downstream_model, report, confuson_matrix
+
+
+def get_probability_cmap():
+    colormap = plt.get_cmap("RdYlGn")
+    cmap = {}
+    for i in range(101):
+        cmap[i] = tuple((np.array(colormap(int(2.55 * i))) * 255).astype(int))
+    return cmap
+
+
+NODATAVALUE = {
+    "cropland": 255,
+    "croptype": 255,
+    "confidence": 255,
+}
+
+
+COLORMAP = {
+    "active-cropland": {
```

Comment:
Why do we need this here?

Reply:
is a remainder from phase I, will delete...